models/staging/commissioning/ers/stg_ers_header.sql

```
@@ -0,0 +1,18 @@
+    rp_start_date,
+    rp_end_date,
+    unique_month_id,
+    total_ebsx02,
+    total_records,
+    dmic_import_log_id,
+    dmic_month_id,
+    dmic_system_id,
+    dmic_ccg_code_referrer,
+    dmic_dscro,
+    dmic_date_added
+from {{ ref('raw_ers_pc_ebsx00header') }}
```

SQLFluff (4.2.2)

[error] Files must end with a single trailing newline.

(LT12)